config/initializers/date_formats.rb

```diff
@@ -1,2 +1,3 @@
 DateTime::DATE_FORMATS[:custom_ordinal] = lambda { |time| time.strftime("%A #{time.day.ordinalize} %B %Y") }
 DateTime::DATE_FORMATS[:govuk_date] = lambda { |time| time.strftime("%d-%m-%Y") }
+DateTime::DATE_FORMATS[:long_date] = lambda { |time| time.strftime("%d %B %Y") }
```

Comment:
```suggestion
DateTime::DATE_FORMATS[:long_date] = lambda { |time| time.strftime("%-d %B %Y") }
```
to stop the leading zero on the day of the month